client/src/views/Register.tsx

```diff
@@ -1,7 +1,71 @@
+import React, { useState } from "react";
+import { useAuthStore } from "../store/authStore";
+import InputField from "../components/InputField";
+import FormError from "../components/FormError";
+
 export default function Register() {
+  const { register } = useAuthStore();
+  const [formData, setFormData] = useState({
+    name: "",
+    email: "",
+    password: "",
+  });
+  const [error, setError] = useState<string | null>(null);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      await register(formData.name, formData.email, formData.password);
+      window.location.href = '/login';
```

Comment:
use `useNavigate` hook for redirect the user, `window.location.href` is depricated.